packages/flutter/test/material/search_anchor_test.dart

```diff
@@ -1187,19 +1187,19 @@ void main() {
       await tester.pumpWidget(buildSearchAnchor(platform));
```

Comment:
It does not look like viewBarPadding is tested?

Reply:
Yep I definitely missed that. I added a test for it.